ffi-module.c

```diff
@@ -989,6 +988,18 @@ init_type_alias (const char *name, bool is_unsigned, int size)
     type_descriptors[i].type = type;
 }
 
+static void
+provide (emacs_env *env, const char *feature)
+{
+  emacs_value Qfeat = env->intern (env, feature);
+  emacs_value Qprovide = env->intern (env, "provide");
+  emacs_value args[] = { Qfeat };
+
+  env->funcall (env, Qprovide, 1, args);
+}
+
+static bool initialized = false;
```

Comment:
I think you probably need `<stdbool.h>` to use `bool` and `false`; but it's just as clear (IMO) with `int` and less finicky.

Reply:
I've done the former. I am not a C programmer and advice on the internets is unclear on what  else to do. I would have used `initialized == 1` but I read that that is a big no no. Please feel free to change this to your preferred style.